src/node_api.cc

```diff
@@ -772,6 +772,11 @@ napi_status napi_get_last_error_info(napi_env env,
   return napi_ok;
 }
 
+NAPI_NO_RETURN void napi_fatal_error(const char* location,
+                                     const char* message) {
+  node::FatalError(location, message);
```

Comment:
Ahh, ok, I hadn't seen that.  It seems like it would be best then to adapt an implementation into node_api.cc that mimics the node::FatalError behavior?